.drone/drone.yml

```diff
@@ -82,7 +82,7 @@ trigger:
 
```

Comment:
Thanks for updating this :) You need to resign the file though: https://github.com/grafana/agent/blob/main/.drone/README.md